components/brave_shields/browser/brave_shields_util.cc

```diff
@@ -106,6 +106,38 @@ ControlType ControlTypeFromString(const std::string& string) {
   }
 }
 
+std::string FingerprintingControlTypeToString(ControlType type) {
+  switch (type) {
+    case ControlType::ALLOW:
+      return "allow";
+    case ControlType::BLOCK:
+      return "block";
+    case ControlType::BLOCK_THIRD_PARTY:
+      FALLTHROUGH;
```

Comment:
doesn't this also need a v2 check?

Reply:
In the UI, same string `default` is only used for `BLOCK_THIRD_PARTY` and `DEFAULT`.